lib/js/dom_extensions.nim

```diff
@@ -0,0 +1,5 @@
+import dom
+
+{.push importcpp.}
```

Comment:
`{.importjs.}` ?. I think it works just Ok now.

Reply:
maybe , as it is a new file, maybe ok to switch to `importjs`